### DISCUSSION TITLE
Rendering 3D objects inside window

### DISCUSSION THREAD

**Discussion by MP185496** [callstack/react-native-visionos#113] [Q&A] [answered]
Is there any way to render 3D objects from RN and interact with it, or we need to do it through swiftUI?

**Comment by okwasniewski** [comment, marked as answer]
Hey, currently there is no built-in way to do it. I have plans for 3rd party module that would support rendering SwiftUI's Model3D view in React Native but I have no ETA at the moment.

**Reply by MP185496** [reply]
Ok, thank you very much
